### DISCUSSION TITLE
Could you tell me how to configure Sparkle/zh_CN.lproj

### DISCUSSION THREAD

**Discussion by duolabmeng6** [sparkle-project/Sparkle#2215] [Q&A]
I'm sorry to ask this question because I checked the documentation and didn't set the option of localizing the language.

**Comment by zorgiepoo** [comment]
Changes to the localizations need to be contributed to Sparkle via pull requests.

If you're asking how to use the localizations that's something Cocoa somehow handles switching when your system language changes.
